packages/wonder-blocks-tooltip/components/tooltip-arrow.js

```diff
@@ -0,0 +1,232 @@
+                };
+
+            default:
+                throw new Error(`Unknown placement: ${placement}`);
+        }
+    }
+
+    _renderArrow() {
+        const {placement} = this.props;
+        const {
+            points,
+            height,
+            width,
+        } = this._calculateDimensionsFromPlacement();
+
+        // TODO(somewhatabstract): Use unique IDs!
+        const dropShadowFilterId = `tooltip-dropshadow-${placement}`;
+
+        return (
+            <svg
+                className={css(styles.arrow, styles[`arrow-${placement}`])}
+                width={width}
+                height={height}
+            >
+                {/* Create an SVG filter that applies a blur to an element.
+                  * We'll apply it to a dark shape outlining the tooltip, which
+                  * will produce the overall effect of a drop-shadow.
+                  *
+                  * Also, scope its ID by side, so that tooltips with other
+                  * "side" values don't end up using the wrong filter from
+                  * elsewhere in the document. (The `height` value depends on
+                  * which way the arrow is turned!)
+                  *
+                  * In general, it's not *great* that multiple tooltips of the
+                  * same side will yield ID conflicts... but at least they'll
+                  * be the same filter, so it doesn't matter which one we
+                  * reference, so long as it's for the correct side. */}
+                <filter id={dropShadowFilterId} height="150%">
+                    <feOffset
+                        dx={Spacing.xSmall}
+                        dy={Spacing.xSmall}
+                        result="offsetblur"
+                    />
+                    <feGaussianBlur
+                        in="SourceAlpha"
+                        stdDeviation={Spacing.xxSmall / 2}
+                    />
+                    <feComponentTransfer>
+                        <feFuncA type="linear" slope={0.16} />
+                    </feComponentTransfer>
+                    <feMerge>
+                        <feMergeNode />
+                        <feMergeNode in="SourceGraphic" />
+                    </feMerge>
+                </filter>
+
+                {/* Draw the background of the tooltip arrow. */}
+                <polyline
+                    fill={Colors.white}
+                    stroke={Colors.white}
+                    points={points.join(" ")}
+                />
+
+                {/* Draw an outline around the tooltip arrow, and apply the
+                  * blur filter we created above, to produce a drop shadow
+                  * effect. */}
+                <polyline
+                    // Redraw the stroke on top of the background color,
+                    // so that the ends aren't extra dark where they meet
+                    // the border of the tooltip.
+                    fill={Colors.white}
```

Comment:
If this also has `fill={Colors.white}` do we need the first polyline?

Reply:
Yes. If we take it away, the drop shadow is much much darker and prominent.

Reply:
Also, because our `offBlack` has opacity, if we don't draw the initial white line, we blend with whatever is under the arrow, rather than white.

Reply:
Interesting... I wonder what this looks like over a dark background.

Reply:
It looks great! ...though the shadow is less visible for obvious reasons.